src/frontend/screens/Login/index.tsx

```diff
@@ -75,13 +75,12 @@ export default React.memo(function NewLogin() {
     navigate('/')
   }
 
+  if (loading) {
+    return <UpdateComponent />
+  }
```

Comment:
this was inside the other `return` value and was rendering the loading along with other elements which looked really broken